app/src/main/java/com/abisoft/todocompose/ui/screens/AddTaskScreen.kt

```diff
@@ -66,7 +84,28 @@ fun AddTaskScreen(onTaskAdded: () -> Unit, onClose: () -> Unit) {
                         Text(
                             text = "СОХРАНИТЬ",
                             color = Color.Blue,
-                            modifier = Modifier.padding(horizontal = 16.dp)
+                            modifier = Modifier
+                                .padding(horizontal = 16.dp)
+                                .clickable {
+                                    val newTodo = TodoDto(
```

Comment:
Maybe you should move padding after clickable (16dp at view corners isn't clickable). Modifiers order is matter